### ISSUE TITLE
[BUG] fvm use on a dart package blocked

### ISSUE THREAD

**Issue by krispypen** [leoafarias/fvm#625]
**Before creating a bug report please make check the following**

- [x] You have read our [FAQ](https://fvm.app/docs/guides/faq)
- [x] If you have used flutter. Please install correctly, run `pub cache repair`. Close the terminal and try again.
- [ ] If you are on Windows. Make sure you are running the terminal as `administrator` or with `developer` permissions.
- [x] Run `fvm doctor` if possible and add the output to the issue.

**Describe the bug**
When having pure dart package which does not depend on flutter... When you run "fvm use" it result in the error:
```
This does not seem to be a Flutter project directory
? Would you like to continue? (y/n) › 
```
You can just say "y" but that does not work on CI tests. 

** Context **
We have multiple packages, some depend on flutter and some not, but we all want to use fvm to have the same setup for all...

**Desktop:**

- OS:  macos
- FVM Version 3.0.3

**Comment by leoafarias** [comment]
@krispypen Can you run just `fvm install`? Why do you need to run `fvm use` on the CI?

**Comment by leoafarias** [comment]
Also relateed to #624

**Comment by leoafarias** [comment]
Closed by #626
